packages/elinks.rb

```diff
@@ -1,40 +1,49 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Elinks < Package
+class Elinks < Meson
   description 'Full-Featured Text WWW Browser'
-  homepage 'http://elinks.or.cz/'
-  version '0.13'
+  homepage 'https://github.com/rkd77/elinks'
+  version '0.16.1.1'
   license 'GPL-2'
   compatibility 'all'
-  source_url 'http://elinks.or.cz/download/elinks-current-0.13.tar.bz2'
-  source_sha256 '03bd8cb436830e89407acbc8d4a736356f3780dc863a2e9df593d6c870d0bb55'
+  source_url 'https://github.com/rkd77/elinks.git'
+  git_hashtag "v#{version}"
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.13_armv7l/elinks-0.13-chromeos-armv7l.tar.zst',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.13_armv7l/elinks-0.13-chromeos-armv7l.tar.zst',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.13_i686/elinks-0.13-chromeos-i686.tar.zst',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.13_x86_64/elinks-0.13-chromeos-x86_64.tar.zst'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.16.1.1_armv7l/elinks-0.16.1.1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.16.1.1_armv7l/elinks-0.16.1.1-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.16.1.1_i686/elinks-0.16.1.1-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.16.1.1_x86_64/elinks-0.16.1.1-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '7478d48810157ace3bf8672f68398ff9ba4fff19f93fde529ebe655ff9c4e2f0',
-     armv7l: '7478d48810157ace3bf8672f68398ff9ba4fff19f93fde529ebe655ff9c4e2f0',
-       i686: '7852bd73a485390c86c0416b28dbfc701dd931f22b0c76fbb478133512b8bd4c',
-     x86_64: '21b68488bdc3dda9286760e087288234ceee0ba147cba8ad8b3b0f67f3205309'
+    aarch64: 'e8672bf1e0e8532672bde9e64d129fdc6942580efab6296f794406edfe3c7b7d',
+     armv7l: 'e8672bf1e0e8532672bde9e64d129fdc6942580efab6296f794406edfe3c7b7d',
+       i686: 'c410b2bf0dbd5a5d914f58d9da41822ef9044153e5cd79e6ec47d3387d49447b',
+     x86_64: 'cbc7f1bd03f93429eb81992dea522584f9fb5b46bc8059c1b43f967a5941501e'
   })
 
-  depends_on 'lzip'
-  depends_on 'openssl' # R
-  depends_on 'zlibpkg' # R
+  depends_on 'brotli' # R
   depends_on 'bzip2' # R
   depends_on 'expat' # R
+  depends_on 'gcc_lib' # R
+  depends_on 'glibc' # R
+  depends_on 'gpgme' => :build
+  depends_on 'gpm' # R
+  depends_on 'lzip' => :build
+  depends_on 'openssl' # R
+  depends_on 'tre' # R
+  depends_on 'zlibpkg' # R
 
-  def self.build
-    system "./configure --without-x #{CREW_OPTIONS}"
-    system 'make'
-  end
+  meson_options '-Dcgi=true \
+    -D256-colors=true \
+    -Dhtml-highlight=true \
+    -Didn=false \
+    -Dgemini=true \
+    -Dbrotli=true \
+    -Dbzlib=true \
+    -Dzlib=true'
```

Comment:
`zlib` is true by default in `0.16.1.1`

https://github.com/rkd77/elinks/blob/v0.16.1.1/meson_options.txt#L44